elastic/kibana
# [Metrics UI] Fix fatal error when no data is available for Metric Threshold Alerts

# Summary

This PR fixes a fatal error with the Metric Threshold Alerts in the central management UI when there is no data available. This also adds i18n support for the error message.

![Jun-09-2020 12-34-07](https://user-images.githubusercontent.com/41702/84181964-84be9c00-aa3e-11ea-885e-750ef9ae58c0.gif)

### Testing
1. Start ES and Kibana without any Metricbeat data
1. Go to central management UI for Alerts & Actions
1. Create a new alert, pick metric threshold alert
1. You should get a message that says "Oops, no chart data available"
